person.rb

```diff
@@ -0,0 +1,22 @@
+# person.rb
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age: 18, name: 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] The parameter `age` should not be an optional parameter, it should be required.

- [ ] **[OPTIONAL]** It would be nice to auto-generate the `id` attribute, instead of passing it as a parameter. This way we won't need to track the previous IDs.

  ```
  def initialize(age:, name: 'Unknown', parent_permission: true)
    @id = rand(1...1000)
  ```